### ISSUE TITLE
Error at Cifar-recipe example

### ISSUE THREAD

**Issue by erogol** [dmlc/mxnet#729]
After I run the model.fit(...) cell I see that error

```
MXNetError: [18:00:59] src/symbol/symbol.cc:121: Symbol.InterShapeKeyword argument name softmax_label not found.
Candidate arguments:
	[0]data
	[1]convolution114_weight
	[2]convolution114_bias
	[3]batchnorm114_gamma
	[4]batchnorm114_beta
	[5]convolution115_weight
	[6]convolution115_bias
	[7]batchnorm115_gamma
	[8]batchnorm115_beta
	[9]convolution116_weight
	[10]convolution116_bias
	[11]batchnorm116_gamma
	[12]batchnorm116_beta
	[13]convolution117_weight
	[14]convolution117_bias
	[15]batchnorm117_gamma
	[16]batchnorm117_beta
	[17]convolution118_weight
	[18]convolution118_bias
	[19]batchnorm118_gamma
	[20]batchnorm118_beta
	[21]convolution119_weight
	[22]convolution119_bias
	[23]batchnorm119_gamma
	[24]batchnorm119_beta
	[25]convolution120_weight
	[26]convolution120_bias
	[27]batchnorm120_gamma
	[28]batchnorm120_beta
	[29]convolution121_weight
	[30]convolution121_bias
	[31]batchnorm121_gamma
	[32]batchnorm121_beta
	[33]convolution122_weight
	[34]convolution122_bias
	[35]batchnorm122_gamma
	[36]batchnorm122_beta
	[37]convolution123_weight
	[38]convolution123_bias
	[39]batchnorm123_gamma
	[40]batchnorm123_beta
	[41]convolution124_weight
	[42]convolution124_bias
	[43]batchnorm124_gamma
	[44]batchnorm124_beta
	[45]convolution125_weight
	[46]convolution125_bias
	[47]batchnorm125_gamma
	[48]batchnorm125_beta
	[49]convolution126_weight
	[50]convolution126_bias
	[51]batchnorm126_gamma
	[52]batchnorm126_beta
	[53]convolution127_weight
	[54]convolution127_bias
	[55]batchnorm127_gamma
	[56]batchnorm127_beta
	[57]convolution128_weight
	[58]convolution128_bias
	[59]batchnorm128_gamma
	[60]batchnorm128_beta
	[61]convolution129_weight
	[62]convolution129_bias
	[63]batchnorm129_gamma
	[64]batchnorm129_beta
	[65]convolution130_weight
	[66]convolution130_bias
	[67]batchnorm130_gamma
	[68]batchnorm130_beta
	[69]convolution131_weight
	[70]convolution131_bias
	[71]batchnorm131_gamma
	[72]batchnorm131_beta
	[73]convolution132_weight
	[74]convolution132_bias
	[75]batchnorm132_gamma
	[76]batchnorm132_beta
	[77]fullyconnected6_weight
	[78]fullyconnected6_bias
	[79]softmaxoutput5_label
```

**Comment by piiswrong** [comment]
The examples under notebooks are outdated. You are welcome to submit a fix or use examples under image-classification. 

If you want to fix this, I suggest changing the name of SoftmaxOutput to softmax

**Comment by antinucleon** [comment]
Please use this example to train CIFAR-Network; https://github.com/dmlc/mxnet/tree/master/example/image-classification

**Comment by wangg12** [comment]
Fixed in PR #737
